### PR TITLE
Introduced BytesFrom

### PR DESCRIPTION
**Pull request by krzysztofdrys**
Introduced BytesFrom for the perfotmant computing distance between bytes slices

### CODE REVIEW

File: bytes.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		return 0
+	}
+	return b.subCost
+}
+
+// BytesFrom computes distance from one byte array to other byte arrays.
+// Instances should not be used concurrently.
+type BytesFrom interface {
+	Dist(to []byte) int
+}
```
Comment (robert-zaremba) [inline bytes.go:47]: why introducing new name? How about `DStr`?

Comment (krzysztofdrys) [reply bytes.go:47]: `D` has a different signature (`Dist(string, string) int` vs `Dist([]byte) int`). I was thinking that we might also have `DBytes` (so `Dist([]byte, []byte)`) and `StringFrom` (so `Dist(string) int`).

Comment (robert-zaremba) [reply bytes.go:47]: Yes, In my comment above I made an error, was thinking about `DBytes` rather then `DStr` (thought `DStr` or `DString` could be used for `D`)

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Pull request merged**